e2e-tests/spring-rest-openapi-v2/src/main/java/com/foo/rest/examples/spring/security/accesscontrol/deleteput/SimpleAccessControlRest.java

```diff
@@ -0,0 +1,150 @@
+package com.foo.rest.examples.spring.security.accesscontrol.deleteput;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+import java.util.HashMap;
+
+/*
+Vulnerable application in which only authorized users can create or delete resources but
+any user can modify resources.
+
+This application was developed to satisfy the request:
+
+delete this file once added example
```

Comment:
can leave comments

Reply:
Hi Andrea, 

Thanks for your comment. I made this comment as:

This application was developed to satisfy the request:

An example application with 2 endpoints: PUT and DELETE on the same resource

/api/{x}